src/Common/MoistThermodynamics/MoistThermodynamics.jl

```diff
@@ -133,34 +129,28 @@ The (moist-)air density from the equation of state
 air_density(ts::ThermodynamicState) = ts.ρ
 
 """
-    specific_volume(T, p[, q::PhasePartition])
+    specific_volume(ts::ThermodynamicState)
 
 The (moist-)air specific volume from the equation of
-state (ideal gas law) where
-
- - `T` air temperature
- - `p` pressure
-and, optionally,
- - `q` [`PhasePartition`](@ref). Without this argument the results are that of dry air.
+state (ideal gas law), given a thermodynamic state `ts`.
```

Comment:
I'd remove ``from the equation of state (ideal gas law)`` here, because typically you have density as a state variable.

Reply:
Implemented in #566